services/showcase/api/db/migrations/versions/0e7a0e8f75ad_add_issuers_bools.py

```diff
@@ -0,0 +1,80 @@
+"""add issuers bools
+
+Revision ID: 0e7a0e8f75ad
+Revises: 3449c639cdbd
+Create Date: 2022-02-25 19:22:56.487174
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "0e7a0e8f75ad"
+down_revision = "3449c639cdbd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
```

Comment:
I would remove anything that you didn't explicitly want - I believe you just want the 3 new columns on tenant.